protos/node/node.proto

```diff
@@ -100,6 +100,7 @@ message StorageStat {
     float score = 3;
     int64 storage_used = 4;
     int64 storage_cap = 5;
+    int64 storage_disk_total = 6;
```

Comment:
do we have any plan to send and store this in status-db? or is it for local use only?